scripts/testing/docker-compose_otel-collector.yaml

```diff
@@ -0,0 +1,73 @@
+version: '3.9'
+
+networks:
+  monitoring:
+    driver: bridge
+
+volumes:
+  prometheus_data: {}
+  victoriametrics_data: {}
+
+services:
+  otel-collector:
+    image: otel/opentelemetry-collector
+    container_name: collector
+    restart: always
+    command: ["--config=/etc/otel-collector-config.yaml"]
+    volumes:
+      - ./otel-collector.yaml:/etc/otel-collector-config.yaml
+    ports:
+      - "1888:1888"   # pprof extension
+      - "13133:13133" # health_check extension
+      - "4317:4317"   # OTLP/2 gRPC receiver
+      - "4318:4318"   # OTLP/2 http receiver
+      - "55679:55679" # zpages extension
+      - "8889:8889"   # prometheus exporter
+    networks:
+      - monitoring
+
+  # grafana:
```

Comment:
is this required? If not, we could add a different compose yaml so we can run it separately?

Reply:
I removed it for now, I'll add a separate docker compose file later (once I'm sure it works).